packages/flutter_tools/lib/src/hot.dart

```diff
@@ -201,29 +199,6 @@ class HotRunner extends ResidentRunner {
       return 1;
     }
 
-    // TODO(devoncarew): We shouldn't have to do type checks here.
-    if (shouldBuild && device is AndroidDevice) {
-      printTrace('Running build command.');
-
-      await buildApk(
-        device.platform,
-        target: target,
-        buildMode: debuggingOptions.buildMode
-      );
-    }
-
-    // TODO(devoncarew): Move this into the device.startApp() impls.
-    if (_package != null) {
-      printTrace("Stopping app '${_package.name}' on ${device.name}.");
-      await device.stopApp(_package);
-    }
-
-    if (prebuiltMode || device is AndroidDevice) {
```

Comment:
FYI, this removal broke prebuilt mode `flutter run` - it no longer installs the app.